portal/templates/dashboard/_macros/dashboard-facilities-table.njk

```diff
@@ -4,135 +4,171 @@
-      {% set value = currency + ' ' + facility.value %}
-    {% endif %} #}
-    {% set currencyAndValue = '-' %}
-
-    {% if facility.currency.id and facility.value %}
-      {% set formattedValue = facility.value | formatAsCurrency %}
-      {% set currencyAndValue = facility.currency.id + ' ' + formattedValue %}
-    {% endif %}
-
-    {% set facilityRow = [
-      {
-        html: linkHtml,
-        attributes: { "data-cy": "facility__bankId--" + facility._id  },
-        classes: "govuk-body-s"
-      },
-      {
-        text: facility.ukefFacilityId | dashIfEmpty,
-        attributes: { "data-cy": "facility__ukefId--" + facility._id },
-        classes: "govuk-body-s"
-      },
-      {
-        text: facility.type | dashIfEmpty,
-        attributes: { "data-cy": "facility__type--" + facility._id },
-        classes: "govuk-body-s"
-      },
-      {
-        text: facility.submissionType | dashIfEmpty,
-        attributes: { "data-cy": "facility__noticeType--" + facility._id },
-        classes: "govuk-body-s"
-      },
-      {
-        text: currencyAndValue,
-        attributes: { "data-cy": "facility__value--" + facility._id },
-        classes: "govuk-body-s"
-      },
-      {
-        text: stage  | dashIfEmpty,
-        attributes: { "data-cy": "facility__bankStage--" + facility._id },
-        classes: "govuk-body-s"
-      },
-      {
-        text: facility.submittedAsIssuedDate | localiseTimestamp("D MMM YYYY", userTimezone) | dashIfEmpty,
-        attributes: { "data-cy": "facility__issuedDate--" + facility._id },
-        classes: "govuk-body-s"
-      }
-    ] %}
-
-    {% set rows = (rows.push(facilityRow), rows) %}
-  {% endfor %}
-
-  {{govukTable({
-    classes: "govuk-table govuk-!-margin-bottom-0",
-    attributes: {
-      'data-module': 'moj-sortable-table'
-    },
-    head: [
-      {
-        text: "Facility name",
-        attributes: {
-          "data-cy": "facility__name--header"
-        }
-      },
-      {
-        text: "UKEF facility ID",
-        attributes: {
-          "data-cy": "facility__ukefId--header"
-        }
-      },
-      {
-        text: "Facility type",
-        attributes: {
-          "data-cy": "facility__type--header"
-        }
-      },
-      {
-        text: "Notice type",
-        attributes: {
-          "data-cy": "facility__noticeType--header"
-        }
-      },
-      {
-        text: "Value",
-        attributes: {
-          "data-cy": "facility__value--header"
-        }
-      },
-      {
-        text: "Bank’s facility stage",
-        attributes: {
-          "data-cy": "facility__bankStage--header"
+      {% set linkHtml %}
+        {% if not isChecker %}
+          <a href="{{ url }}" class="govuk-link" data-cy="facility__name--link--{{ facility._id }}">
+            {{ facility.name if facility.name else 'Not entered'}}
+          </a>
+        {% else %}
+          <span data-cy="facility__name--text--{{ facility._id }}"> {{ facility.name }} </span>
```

Comment:
Leading and Trailing white spaces are required, is it to satisfy the design?
If yes then please ignore this comment.

Reply:
done, rest is kept constant with rest of file, unless you meant whitespace elsewhere

Reply:
done